llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -2566,9 +2566,9 @@ static Value *getExpandedStep(const InductionDescriptor &ID,
     return C->getValue();
   if (auto *U = dyn_cast<SCEVUnknown>(Step))
     return U->getValue();
-  auto I = ExpandedSCEVs.find(Step);
-  assert(I != ExpandedSCEVs.end() && "SCEV must be expanded at this point");
-  return I->second;
+  auto *V = ExpandedSCEVs.lookup(Step);
```

Comment:
```suggestion
   Value *V = ExpandedSCEVs.lookup(Step);
```

now we can spell out the simple type